integrationtesting/opendcs-tests/src/test/resources/data/cwms/cwms_location_level_test_data.sql

```diff
@@ -57,65 +57,177 @@ BEGIN
-        DBMS_OUTPUT.PUT_LINE('Created CACHE_TEST location');
+        DBMS_OUTPUT.PUT_LINE('Created YETL-Surface location');
     EXCEPTION
         WHEN OTHERS THEN
-            DBMS_OUTPUT.PUT_LINE('Cache test location may already exist: ' || SQLERRM);
+            DBMS_OUTPUT.PUT_LINE('YETL-Surface location may already exist: ' || SQLERRM);
     END;
     
     -- Store location levels using the simpler STORE_LOCATION_LEVEL procedure
+    -- Create multiple values over time for range testing
     BEGIN
-        -- Stage level for TEST_LOCATION
+        -- Stage level for FTPK-Lower - multiple values over time
+        -- Current value
         CWMS_20.CWMS_LEVEL.STORE_LOCATION_LEVEL(
-            P_LOCATION_LEVEL_ID => 'TEST_LOCATION.Stage.Const.0.Test',
+            P_LOCATION_LEVEL_ID => 'FTPK-Lower.Stage.Const.0.Test',
             P_LEVEL_VALUE      => 123.45,
             P_LEVEL_UNITS      => 'ft',
             P_EFFECTIVE_DATE   => SYSDATE,
             P_TIMEZONE_ID      => 'UTC',
             P_OFFICE_ID        => v_office_id
         );
-        DBMS_OUTPUT.PUT_LINE('Created TEST_LOCATION.Stage.Const.0.Test level');
+        DBMS_OUTPUT.PUT_LINE('Created FTPK-Lower.Stage.Const.0.Test level at current time');
+        
+        -- Value from 1 day ago
+        CWMS_20.CWMS_LEVEL.STORE_LOCATION_LEVEL(
+            P_LOCATION_LEVEL_ID => 'FTPK-Lower.Stage.Const.0.Test',
+            P_LEVEL_VALUE      => 122.50,
+            P_LEVEL_UNITS      => 'ft',
+            P_EFFECTIVE_DATE   => SYSDATE - 1,
+            P_TIMEZONE_ID      => 'UTC',
+            P_OFFICE_ID        => v_office_id
+        );
+        DBMS_OUTPUT.PUT_LINE('Created FTPK-Lower.Stage.Const.0.Test level at -1 day');
+        
+        -- Value from 7 days ago
+        CWMS_20.CWMS_LEVEL.STORE_LOCATION_LEVEL(
+            P_LOCATION_LEVEL_ID => 'FTPK-Lower.Stage.Const.0.Test',
+            P_LEVEL_VALUE      => 121.00,
+            P_LEVEL_UNITS      => 'ft',
+            P_EFFECTIVE_DATE   => SYSDATE - 7,
+            P_TIMEZONE_ID      => 'UTC',
+            P_OFFICE_ID        => v_office_id
+        );
+        DBMS_OUTPUT.PUT_LINE('Created FTPK-Lower.Stage.Const.0.Test level at -7 days');
+        
+        -- Value from 14 days ago
+        CWMS_20.CWMS_LEVEL.STORE_LOCATION_LEVEL(
+            P_LOCATION_LEVEL_ID => 'FTPK-Lower.Stage.Const.0.Test',
+            P_LEVEL_VALUE      => 120.25,
+            P_LEVEL_UNITS      => 'ft',
+            P_EFFECTIVE_DATE   => SYSDATE - 14,
+            P_TIMEZONE_ID      => 'UTC',
+            P_OFFICE_ID        => v_office_id
+        );
+        DBMS_OUTPUT.PUT_LINE('Created FTPK-Lower.Stage.Const.0.Test level at -14 days');
+        
+        -- Value from 21 days ago
+        CWMS_20.CWMS_LEVEL.STORE_LOCATION_LEVEL(
+            P_LOCATION_LEVEL_ID => 'FTPK-Lower.Stage.Const.0.Test',
+            P_LEVEL_VALUE      => 119.75,
+            P_LEVEL_UNITS      => 'ft',
+            P_EFFECTIVE_DATE   => SYSDATE - 21,
```

Comment:
don't worry about it in this one, for now, but in the future favor full specific date times.